react/features/chat/components/web/ChatHeader.js

```diff
@@ -46,9 +46,12 @@ function Header({ onCancel, className, isPollsEnabled, t }: Props) {
 
     return (
         <div
-            className = { className || 'chat-dialog-header' }
-            role = 'heading'>
-            { t(isPollsEnabled ? 'chat.titleWithPolls' : 'chat.title') }
+            className = { className || 'chat-dialog-header' }>
+            <span
+                aria-level = { 1 }
+                role = 'heading'>
+                { t(isPollsEnabled ? 'chat.titleWithPolls' : 'chat.title') }
+            </span>
```

Comment:
The idea of removing the `role="heading"` of the `div` and adding back on this new `span`, is to separate the actual heading text from the action button. It doesn't make much sense for a screen reader user to hear the action button text when going through headings.